docs/docsite/rst/administration/troubleshooting.rst

```diff
@@ -8,6 +8,21 @@ Troubleshooting AWX
    single: troubleshooting
    single: help
   
+
+Some troubleshooting tools are built in the AWX user interface that may help you address some issues you might encounter. To access these tools, navigate to **Settings** and select **Troubleshooting**.
+
+.. image:: ../common/images/settings_troubleshooting_highlighted.png
+
+The options available are:
+
+- **Enable or Disable tmp dir cleanup**: choose whether you want to clean up the ``temp`` directory.
```

Comment:
Not familiar with AWX but the UI calls it the tmp dir, so should the text say ``tmp`` directory here as well?